.vsts-ci/templates/releaseBuildPhase.yml

```diff
@@ -20,12 +20,9 @@ jobs:
     displayName: Get Latest ${{ parameters.channel }} Release Tag
     condition: and(succeeded(), not(startsWith(variables['releaseTag'], 'v')))
 
-  - task: PowerShell@2
+  - powershell: |
+      ./tools/releaseBuild/vstsbuild.ps1 -ReleaseTag $(releaseTag) -Name powershell-snap-${{ parameters.channel }}
```

Comment:
the build name varies because they require different images and I forgot to make this change